pkg/eval/unix/umask.go

```diff
@@ -70,22 +74,22 @@ func (UmaskVariable) Set(v interface{}) error {
 		if err != nil {
 			i, err = strconv.ParseInt(v, 0, 0)
 			if err != nil {
-				return errors.New("umask value not a valid number")
+				return ErrUmaskInvalid
 			}
 		}
 		umask = int(i)
 	case float64:
 		intPart, fracPart := math.Modf(v)
 		if fracPart != 0 {
-			return errors.New("umask value must be an integer")
+			return ErrUmaskInvalid
 		}
 		umask = int(intPart)
 	default:
-		return errors.New("umask value must be a string or float64")
+		return ErrUmaskInvalid
 	}
 
 	if umask < 0 || umask > 0o777 {
-		return errors.New("umask value outside the range [0..0o777]")
+		return ErrUmaskInvalid
```

Comment:
Returning the same error gives less information to the user. Use errs.BadValue and errs.OutOfRange to build the error.

Reply:
Done.